reference-architecture/osp-cli/ch4.8.4_install_docker_nodes.sh

```diff
@@ -0,0 +1,4 @@
+for H in $NODES
+do
+  ssh $H sudo yum -y install docker
```

Comment:
Oh wait.  No this is because it is configuring docker storage on the nodes.  Docker must be running after configuring the storage and before running docker-storage-config.